lib/lytics/index.js

```diff
@@ -62,7 +63,10 @@ Lytics.prototype.loaded = function(){
  */
 
 Lytics.prototype.page = function(page){
-  window.jstag.send(page.properties());
+  var props = page.properties();
+  props._e = props.name;
+  delete props.name;
```

Comment:
whoa sorry, that comment went through way before I finished editing! 

this was the snippet i meant to send:

```
var props = page.properties({ name: '_e' });
```

the `.properties()` method does return a clone, so it's definitely safe. as for the `name` -> `_e` conversion, the idiomatic way is just to pass the alias into the `.properties()` method, which will alias anything matching the `.[<aliasKey>]()` [facade](https://github.com/segmentio/facade/) method (which is case insensitive, etc).

https://github.com/segmentio/facade/blob/master/lib/track.js#L164-L185 

of course, that alias *does* use delete... but that's on us and will get swept up in a forthcoming refactor :)

sorry for the confusion!